File: back/Paper_back/Paper_back/Models/Prod.cs
namespace Paper_back.Models
{
    public class Product
    {
        public int Id { get; set; }
        public string Name { get; set; } = "Nouveau produit"; // Valeur par défaut
        public string Texture { get; set; } = string.Empty; // Valeur par défaut
        public int Grammage { get; set; }
        public string Couleur { get; set; } = string.Empty; // Valeur par défaut
    }
}

File: front/src/app/models/prod.ts
export class Product {
  public id: number;
  public name = 'Nouveau produit';
  public texture = '';
  public grammage :number;;
  public couleur = '';

  constructor(id?: number , grammage?: number) {
   
    this.id = id || 0;
    this.grammage = grammage || 0;
  }
}
